src/views/About/CommunityPage.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <div class="flex justify-center text-center flex-col m-16 uppercase text-xl">
+    <img
+      :src="logo"
+      alt="logo"
+      class="mx-auto w-72"
+    >
+    <br>
+    <p>
+      As a DAO, we are encouraging decentralized contributions to <br>
+      the project. Join the Project by contacting us on Discord.<br>
+      <b> Our Community Contributors </b>
+    </p>
+    <br>
+    <p v-for="name in names">
+      {{ name }}
+    </p>
+  </div>
+</template>
+<script setup lang="ts">
+import logo from "@/assets/figma/CCLogo.png";
+
+const names = [
+  "RUSLAN",
+  "“Vnode_casH”",
+  "Carden",
+  "cryptobtcbuyer",
+  "MMS",
+  "BonyNode",
+  "safe",
+  "sminempepe#7667 (@sminempepe)",
+  "Gliderso",
+  "Tarzik",
+  "ArtiDS_Node",
+  "Cardep",
+  "stakr.space",
+  "Shoni",
+  "Yurbason",
+  "icodragon [NODERS]#4560",
+  "JahaND",
+  "gönlü zengin#8838",
+  "xAlex",
+  "Sr20de",
+  "Oxyde",
+  "Asadovka | Nodejumper",
+  "[NODERS]TEAM",
+  "molla202",
+  "dimasik",
+  "kjnodes",
+  "Cumulo",
+  "Nodes|Bull Rancho",
+  "NodeStake",
+  "Mondrago#9340 (@Mr__)",
+  "Apollo",
+  "dreamstaker",
+  "Lis1",
+  "ser#8113, serotonin",
+  " 79anvi ",
+  "ArcLing",
+  "cryptech",
+  "Anatolian Team",
+  "ozzycard",
+  "AlxVoy  ANODE.TEAM",
+  "ITpro",
+  "SINNER",
+  "Biker4b",
+  "sashamaxymchuk",
+  "itrocket",
+  "bulevar",
+  "Nodeist",
+  "laserlincoln#3357",
+  "cardex",
+  "cardchain.acloud.pp.ua",
+  "⭐️ PPNV Service ⭐️",
+  "D-stake",
+  "AnnaSot",
+  "Decloud Nodes Lab",
+  "Validator.run",
+  "@JulyC92894454",
+  "superjax",
+  "Carder",
+  "web34ever",
+  "lesnik | UTSA",
+  "STAVR",
+  "КРИПТОГЕНОФФ",
+  "lesya",
+  "ramuchi.tech",
+  "OranG3cluB",
+  "raymondctc#6331 (@holdnostake)",
+  "AM Solutions ",
+  "078F5410FD08B19E",
+  "KolTigin",
+  "Cardel",
+  "@vadyhodler777",
+  "silent",
+  "ark33",
+  "NOXU",
+  "Moonbridge",
+  "liveevilll",
+  "AlexeyM#5409",
+  "SRG0Z10",
+  "Kolot#5637",
+  "Midora",
```

Comment:
maybe we should ask those guys if it is ok for them to appear on the page, maybe we should strip off the #1234